test/authentication/passwordless.test.js

```diff
@@ -174,7 +174,8 @@ describe('auth0.authentication', function() {
         clientID: '...',
         redirectUri: 'http://page.com/callback',
         responseType: 'code',
-        _sendTelemetry: false
+        _sendTelemetry: false,
+        scope: ''
```

Comment:
I know this is a test. But under what scenario would you want a token without any scope? 🤔

Reply:
I think the point is that it's overridden, but yes we could use a real value in here. Will change